pegasus/sites.v3/code.org/public/css/csjourneys.css

```diff
@@ -0,0 +1,37 @@
+.csjourneys-banner-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  position: relative;
+  padding: 0 10px;
+}
+
+.csjourneys-logo {
+  width: 300px;
+  max-width: 100%;
+  margin-top: 35px;
+  z-index: 1;
+}
+
+h1.csjourneys-banner-header {
+  font-family: "Gotham 4r", sans-serif;
+  color: #fff;
+  font-size: 18px;
+  z-index: 1;
+  margin: 35px 0;
+  text-align: center;
+}
+
+@media only screen and (min-width: 971px) {
```

Comment:
It looks like we use a breakpoint of 960 on some other pages.  Did we intentionally want to use a different breakpoint here?

Reply:
When I was playing with the CS journeys page, this was the point at which the content changed to desktop view